src/api/spoonacular.py

```diff
@@ -474,6 +474,55 @@ def search_ingredients(
     return ingredients
 
 
+def get_recommended_recipes(
+    offset: int = 0,
+    limit: int = 10,
+) -> list:
+    """
+    Returns a list of randomly recommended recipes.
+
+    Args:
+        Number as the number of desired recipe results
```

Comment:
This is not descriptive of what arguments this function accepts
```suggestion
        offset (int): The offset into the search results to start at.
        limit (int): The maximum number of search results to retrieve.
```